### PR TITLE
Clarify bootstrap command

### LINKED ISSUE #160: clarify output of bootstrap command
The _bootstrap_ command's output should be clearer. Here's an example of the current output:

```
The current template configuration is:
name: test-bootstrap
repository: git@github.com:mykmelez/test-bootstrap.git
description: A template app bootstrapped with oghliner.
license: Apache-2.0
Would you like to change any of the above configuration values? yes
Project name: test-bootstrap
Repository URL: git@github.com:mykmelez/test-bootstrap.git
Description: A template app bootstrapped with oghliner.
License: Apache-2.0
[15:36:42] [conflict] Creating README.md
[15:36:42] [conflict] Creating app
[15:36:42] [conflict] Creating .gitignore
[15:36:42] [conflict] Creating gulpfile.js
[15:36:42] [conflict] Creating package.json
[15:36:42] [conflict] Creating app/favicon.ico
[15:36:42] [conflict] Creating app/fonts
[15:36:42] [conflict] Creating app/images
[15:36:42] [conflict] Creating app/index.html
[15:36:42] [conflict] Creating app/robots.txt
[15:36:42] [conflict] Creating app/scripts
[15:36:42] [conflict] Creating app/styles
[15:36:42] [conflict] Creating app/images/apple-touch-icon-114x114.png
[15:36:42] [conflict] Creating app/images/apple-touch-icon-120x120.png
[15:36:42] [conflict] Creating app/images/apple-touch-icon-144x144.png
[15:36:42] [conflict] Creating app/images/apple-touch-icon-152x152.png
[15:36:42] [conflict] Creating app/images/apple-touch-icon-57x57.png
[15:36:42] [conflict] Creating app/images/apple-touch-icon-60x60.png
[15:36:42] [conflict] Creating app/images/apple-touch-icon-72x72.png
[15:36:42] [conflict] Creating app/images/apple-touch-icon-76x76.png
[15:36:42] [conflict] Creating app/images/bg_hr.png
[15:36:42] [conflict] Creating app/images/blacktocat.png
[15:36:42] [conflict] Creating app/images/favicon-128x128.png
[15:36:42] [conflict] Creating app/images/favicon-16x16.png
[15:36:42] [conflict] Creating app/images/favicon-196x196.png
[15:36:42] [conflict] Creating app/images/favicon-32x32.png
[15:36:42] [conflict] Creating app/images/favicon-96x96.png
[15:36:42] [conflict] Creating app/images/icon_download.png
[15:36:42] [conflict] Creating app/images/mstile-144x144.png
[15:36:42] [conflict] Creating app/images/mstile-150x150.png
[15:36:42] [conflict] Creating app/images/mstile-310x150.png
[15:36:42] [conflict] Creating app/images/mstile-310x310.png
[15:36:42] [conflict] Creating app/images/mstile-70x70.png
[15:36:42] [conflict] Creating app/images/sprite_download.png
[15:36:42] [conflict] Creating app/scripts/main.js
[15:36:42] [conflict] Creating app/scripts/offline-manager.js
[15:36:42] [conflict] Creating app/styles/github-light.css
[15:36:42] [conflict] Creating app/styles/stylesheet.css

> ursa@0.9.1 install /Users/myk/Projects/test-bootstrap/node_modules/oghliner/node_modules/travis-encrypt/node_modules/ursa
> node-gyp rebuild

  CXX(target) Release/obj.target/ursaNative/src/ursaNative.o
  CXX(target) Release/obj.target/ursaNative/src/asprintf.o
  SOLINK_MODULE(target) Release/ursaNative.node
gulp@3.9.0 node_modules/gulp
├── interpret@0.6.6
├── pretty-hrtime@1.0.1
├── deprecated@0.0.1
├── archy@1.0.0
├── minimist@1.2.0
├── tildify@1.1.2 (os-homedir@1.0.1)
├── v8flags@2.0.10 (user-home@1.1.1)
├── semver@4.3.6
├── chalk@1.1.1 (escape-string-regexp@1.0.3, supports-color@2.0.0, ansi-styles@2.1.0, strip-ansi@3.0.0, has-ansi@2.0.0)
├── orchestrator@0.3.7 (stream-consume@0.1.0, sequencify@0.0.7, end-of-stream@0.1.5)
├── liftoff@2.2.0 (extend@2.0.1, rechoir@0.6.2, flagged-respawn@0.3.1, resolve@1.1.6, findup-sync@0.3.0)
├── gulp-util@3.0.7 (array-differ@1.0.0, array-uniq@1.0.2, lodash._reevaluate@3.0.0, fancy-log@1.1.0, lodash._reinterpolate@3.0.0, beeper@1.1.0, lodash._reescape@3.0.0, object-assign@3.0.0, replace-ext@0.0.1, has-gulplog@0.1.0, gulplog@1.0.0, vinyl@0.5.3, lodash.template@3.6.2, through2@2.0.0, multipipe@0.1.2, dateformat@1.0.11)
└── vinyl-fs@0.3.14 (graceful-fs@3.0.8, strip-bom@1.0.0, defaults@1.0.3, vinyl@0.4.6, mkdirp@0.5.1, through2@0.6.5, glob-stream@3.1.18, glob-watcher@0.0.6)

gulp-connect@2.2.0 node_modules/gulp-connect
├── connect-livereload@0.3.2
├── event-stream@3.1.7 (stream-combiner@0.0.4, duplexer@0.1.1, from@0.1.3, pause-stream@0.0.11, map-stream@0.1.0, split@0.2.10, through@2.3.8)
├── tiny-lr@0.0.7 (debug@0.8.1, faye-websocket@0.4.4, noptify@0.0.3, qs@0.5.6)
├── gulp-util@2.2.20 (lodash._reinterpolate@2.4.1, minimist@0.2.0, chalk@0.5.1, vinyl@0.2.3, through2@0.5.1, lodash.template@2.4.1, multipipe@0.1.2, dateformat@1.0.11)
└── connect@2.17.3 (parseurl@1.0.1, response-time@1.0.0, cookie@0.1.2, connect-timeout@1.1.0, debug@0.8.1, pause@0.0.1, cookie-signature@1.0.3, vhost@1.0.0, fresh@0.2.2, qs@0.6.6, on-headers@0.0.0, basic-auth-connect@1.0.0, bytes@1.0.0, morgan@1.1.1, serve-favicon@2.0.0, errorhandler@1.0.1, cookie-parser@1.1.0, method-override@1.0.2, body-parser@1.2.2, csurf@1.2.0, compression@1.0.2, express-session@1.2.1, type-is@1.2.0, serve-static@1.1.0, serve-index@1.0.3, multiparty@2.2.0)

oghliner@0.9.3 node_modules/oghliner
├── gulp-rename@1.2.2
├── mozilla-tabzilla@0.5.1
├── gitconfiglocal@1.0.0 (ini@1.3.4)
├── commander@2.9.0 (graceful-readlink@1.0.1)
├── github-slug@1.0.0 (git-remote-origin-url@1.1.2, github-url-from-git@1.4.0)
├── gulp-install@0.6.0 (which@1.2.0)
├── promisified-promptly@1.0.0 (promptly@0.2.1)
├── through2@2.0.0 (xtend@4.0.1, readable-stream@2.0.4)
├── promisify-node@0.2.1 (nodegit-promise@3.0.3)
├── rimraf@2.4.3 (glob@5.0.15)
├── sw-precache@2.2.0 (pretty-bytes@2.0.1, es6-promise@3.0.2, lodash.template@3.6.2, lodash.defaults@3.1.2, dom-urls@1.1.0, glob@5.0.15, meow@3.5.0)
├── gulp-util@3.0.7 (array-differ@1.0.0, array-uniq@1.0.2, fancy-log@1.1.0, lodash._reevaluate@3.0.0, lodash._reinterpolate@3.0.0, beeper@1.1.0, lodash._reescape@3.0.0, object-assign@3.0.0, replace-ext@0.0.1, has-gulplog@0.1.0, minimist@1.2.0, chalk@1.1.1, gulplog@1.0.0, vinyl@0.5.3, lodash.template@3.6.2, multipipe@0.1.2, dateformat@1.0.11)
├── gulp-gh-pages@0.5.4 (readable-stream@2.0.4, gift@0.6.1, vinyl-fs@2.2.1, wrap-promise@1.0.1)
├── github@0.2.4 (mime@1.3.4)
├── gh-pages@0.5.0 (graceful-fs@2.0.1, async@0.2.9, wrench@1.5.1, q@1.0.1, commander@2.8.1, q-io@1.11.6, lodash@2.4.2, glob@4.0.6)
├── travis-ci@2.0.3 (underscore.string@2.2.1, github@0.1.16, lodash@1.3.1, request@2.22.0)
├── gulp-template@3.0.0 (through2@0.6.5, lodash@3.10.1)
├── read-yaml@1.0.0 (xtend@4.0.1, js-yaml@3.4.3)
├── write-yaml@0.2.1 (extend-shallow@1.1.4, write@0.2.1, js-yaml@3.4.3)
├── gulp-conflict@0.4.0 (diff@1.4.0, through2@0.6.5, inquirer@0.8.5)
└── travis-encrypt@1.1.2 (deep-property@1.1.0, colors@0.6.2, yargs@1.3.3, write-yaml@0.1.2, ursa@0.9.1)
```

Next I'll propose a redesign.

### PR DESCRIPTION
**Pull request by marco-c**
Fixes #160.

### CODE REVIEW

**Comment by marco-c** [comment]
Filed #169 to use gulp-install when (and if) https://github.com/slushjs/gulp-install/pull/32 is merged.

**Comment by marco-c** [comment]
First part of the output:
bootstrap_initial

Error while installing npm dependencies:
bootstrap_error

I know the error is logged twice, but there's #150 about it (and I didn't want to suppress errors because it could have unintended consequences. Better to log it twice than never).

Success:
bootstrap_success

File: lib/bootstrap.js [hunk 1]
```diff
@@ -133,11 +141,42 @@ module.exports = function(config) {
...
+            cli.spinner(chalk.green('✓ ') + 'Installing npm dependencies… done!\n', true);
+          })
           .pipe(sink()); // Sink is required to trigger the finish event with install.
-        stream.on('finish', resolve);
+
+        stream.on('finish', function() {
+          console.log(
+            'Your app has been bootstrapped! Just commit the changes and push the commit\n' +
+            'to the origin/master branch:\n\n' +
+            chalk.bold('git commit -m"initial version of Oghliner app" --all') + '\n' +
```
Comment (mykmelez) [inline lib/bootstrap.js:170]: Per that issue @wfwalker ran into with _configure_, I guess this should be `git add --all && git commit -m"initial version of Oghliner app"`.

**Comment by mykmelez** [comment]
In the interest of landing this to unblock other pull requests, I corrected https://github.com/mozilla/oghliner/pull/168#discussion_r44716605 in 5f93c0d7fe513b6fac60373927d289a528a2a943 and merged this manually.

**Pull request merged**